src/components/v5/common/CompletedAction/partials/PaymentBuilder/partials/FundingModal/FundingModal.tsx

```diff
@@ -81,26 +186,33 @@ const FundingModal: FC<FundingModalProps> = ({
 
       await fundExpenditure(payload);
 
-      setIsSubmitting(false);
       onSuccess();
       onClose();
     } catch (err) {
-      setIsSubmitting(false);
       onClose();
     }
   };
 
+  const validationSchema = getValidationSchema(selectedTeam?.id, colony);
```

Comment:
This is a string database ID of format `colonyAddress_nativeId`, in validation schema it's then passed to `hasEnoughFundsValidation` which is actually expecting a native ID instead. I think this is causing the validation to always fail

It would be great if you could update the relevant types from `string` to `number`